packages/zoe/src/contractFacet/internal-types.js

```diff
@@ -20,7 +20,7 @@
 
 /**
  * @typedef ZCFZygote
- * @property {(bundleOrBundleCap: SourceBundle | BundleCap) => void} evaluateContract
+ * @property {(bundleOrBundleCap: Record<string, string>) => void} evaluateContract
```

Comment:
`SourceBundle` wasn’t defined any more specifically than this. It probably should be `unknown` since bundles get trafficked as JSON and probably should be verified anywhere they’re bought or sold. I could easily be swayed to go the other way, making these type unions hyper explicit, but that seemed out of scope for this change.

Reply:
I was swayed and the types have been restored.